orchagent/nexthopkey.h

```diff
@@ -98,6 +117,73 @@ struct NextHopKey
     {
         return (ip_address.getV4Addr() == 0);
     }
+
+    bool isMplsNextHop() const
+    {
+        return (label_stack.getSize() > 0);
+    }
+
+    std::string parse_mpls(const std::string& str)
```

Comment:
suggest renaming the API, API seems to parse the next hop string. May be "parseNexthop". Use camel case convention.

Reply:
Yes, it parses the next hop string, but only the MPLS component.

Reply:
Done:  Function is renamed parseMplsNextHop().